src/App/Utilities/Helpers.cs

```diff
@@ -73,6 +73,17 @@ public static string GetEmptyTableSectionTitle()
 
             return " ";
         }
+
+        public static string LoginImage()
```

Comment:
I'd rather not have a helper for every type of image. Just create one method and pass the icon image string to it via parameter.

Reply:
I'm not completely sure what you mean